ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-main.yaml

```diff
@@ -123,6 +123,25 @@ tests:
         requests:
           cpu: 100m
     workflow: optional-operators-ci-azure
+- as: e2e-azure-private-operator
+  steps:
+    cluster_profile: azure4
+    dependencies:
+      OO_INDEX: ci-index-external-dns-operator-bundle
+    env:
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: external-dns-operator
+      OO_PACKAGE: external-dns-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: export AZURE_PRIVATE_DNS="true" && make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
```

Comment:
Can this be added to already existing `e2e-azure-operator` step? This would save us 1 cluster for each run.

Reply:
yes, but if we add in `e2e-azure-operator` then we have to take segregation in our code to execute both privateDNS and Normal.
other option thinking : `make test-e2e && export AZURE_PRIVATE_DNS="true" && make test-e2e`

Reply:
Had quick check wtih Andrey,  we can add as step as below 
- as: test
      cli: latest
      commands: export AZURE_PRIVATE_DNS="true" && make test-e2e
      from: src
      resources:
        requests:
          cpu: 100m

Reply:
I will make a new PR with this change